### ISSUE TITLE
Yet again some weird error: Cannot find module './document'

### ISSUE THREAD

**Issue by Draghmar** [sindresorhus/sublime-autoprefixer#89]
I think it happened after ST updated to the 4175 version. Now trying to use Autoprefixer gives the result below. I did try to update from within package folder.

```
---------------------------
Sublime Text
---------------------------
Autoprefixer
Error: node:internal/modules/cjs/loader:1039

  const err = new Error(message);

              ^



Error: Cannot find module './document'

Require stack:

- H:\Sublime Text\Data\Packages\Autoprefixer\node_modules\postcss\lib\lazy-result.js

- H:\Sublime Text\Data\Packages\Autoprefixer\node_modules\postcss\lib\postcss.js

    at Module._resolveFilename (node:internal/modules/cjs/loader:1039:15)

    at Module._load (node:internal/modules/cjs/loader:885:27)

    at Module.require (node:internal/modules/cjs/loader:1105:19)

    at require (node:internal/modules/cjs/helpers:103:18)

    at Object.<anonymous> (H:\Sublime Text\Data\Packages\Autoprefixer\node_modules\postcss\lib\lazy-result.js:7:16)

    at Module._compile (node:internal/modules/cjs/loader:1218:14)

    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)

    at Module.load (node:internal/modules/cjs/loader:1081:32)

    at Module._load (node:internal/modules/cjs/loader:922:12)

    at Module.require (node:internal/modules/cjs/loader:1105:19) {

  code: 'MODULE_NOT_FOUND',

  requireStack: [

    'H:\\Sublime Text\\Data\\Packages\\Autoprefixer\\node_modules\\postcss\\lib\\lazy-result.js',

    'H:\\Sublime Text\\Data\\Packages\\Autoprefixer\\node_modules\\postcss\\lib\\postcss.js'

  ]

}



Node.js v19.3.0
```

**Comment by samuelcerezo** [comment]
+1 with Node.js 20.10.0. I've updated node (22.1.0) without luck.

**Comment by sindresorhus** [comment]
Should be fixed in 2.0.2.
